cluster/gce/util.sh

```diff
@@ -2066,20 +2066,16 @@ function create-node-template() {
     do
       IFS="," read -r -a ssdopts <<< "${ssdgroup}"
       validate-node-local-ssds-ext "${ssdopts}"
-      for i in $(seq ${ssdopts[0]}); do
+      for ((i=1; i<=ssdopts[0]; i++)); do
         local_ssds="$local_ssds--local-ssd=interface=${ssdopts[1]} "
       done
     done
   fi
 
-  if [[ ! -z ${NODE_LOCAL_SSDS+x} ]]; then
-    # The NODE_LOCAL_SSDS check below fixes issue #49171
```

Comment:
Can we keep the comment so someone doesn't re-introduce the bug when refactoring?

Reply:
Yes, I'll fix it immediately!